src/App.js

```diff
@@ -4,27 +4,61 @@ import React, { useState, useEffect } from 'react';
 import axios from 'axios';
 
 
+
 function App() {
   const [fighters, setFighters] = useState([]);
  
-  React.useEffect(() => {
+      useEffect(() => {
     axios.get('https://api.octagon-api.com/fighters').then((response) => {
       setFighters(response.data);
     });
 
   }, []);
-  {Object.values(data).map(fighter => fighter.name)};
-  console.log(fighters);
-  <ul>
-  
-   
-  <li key={fighters.name}>{fighters.name}</li>
-
-  </ul>
-  
-  console.log(fighters);
 
+return(
+    <div>
+     
+     <table
```

Comment:
I was think maybe you could move some of these logic into a new component such as:

components/FightersList.tsx